db/schema.rb

```diff
@@ -103,10 +103,11 @@
 
   create_table "vendors", force: :cascade do |t|
     t.string   "name"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at",                      null: false
+    t.datetime "updated_at",                      null: false
     t.string   "slug"
     t.string   "about"
+    t.string   "status",     default: "Inactive"
```

Comment:
this is great. I like this idea of aways saving businesses and defaulting them to inactive. 

Do we want to make this an enum?  Also should we have a new status for new businessrequests? Instead of lumping them into inactive? maybe (Active, New, Inactive)?